web/js/map/ui.js

```diff
@@ -853,7 +853,7 @@ export function mapui(models, config) {
         return;
       }
       pixels = map.getEventPixel(e.originalEvent);
-      coords = map.getCoordinateFromPixel(pixels);
+      coords = map.getCoordinateFromPixel(pixels) || [0, 0];
```

Comment:
I don't fully understand this code, but I think it might be better to add a line after this like;
```
if (!coords) return;
```
Rather than setting a coordinate of `[0,0]`.

Reply:
Good eye, thanks